bin/crucible

```diff
@@ -53,6 +53,17 @@ function setup_pipe() {
     mknod ${PIPE} p
 }
 
+if [ "${1}" != "update" ]; then
+    ${CRUCIBLE_HOME}/bin/_git-fetch &
+    if [ -e "${UPDATE_STATUS_FILE}" ]; then
+        read OTHER UPDATE_COUNT < "${UPDATE_STATUS_FILE}"
```

Comment:
what is this magic around OTHER ?

Reply:
I debated this when I originally wrote it.  The file ends up having two values written to it, a timestamp and an update count.  In this instance I'm just reading the update count and don't care about the value in the timestamp field so I just made up an arbitrary variable name to stuff that timestamp value into.  I can make the variable name more accurate if you think it makes more sense that way.